src/hitchhiker/tree/bootstrap/konserve.cljc

```diff
@@ -121,54 +120,57 @@
   ;; TODO check whether store is using nippy in the future and load on the fly:
   #_[hitchhiker.tree.codec.nippy :as nippy]
   #_(nippy/ensure-installed!)
-  (swap! (:read-handlers store)
-         merge
-         {'hitchhiker.tree.bootstrap.konserve.KonserveAddr
-          (fn [{:keys [last-key konserve-key]}]
-            (konserve-addr store
-                           last-key
-                           konserve-key))
-          'hitchhiker.tree.DataNode
-          (fn [{:keys [children cfg] :as d}]
-            (tree/data-node (into (sorted-map-by c/-compare)
-                                  children)
-                            cfg))
-          'hitchhiker.tree.IndexNode
-          (fn [{:keys [children cfg op-buf]}]
-            (tree/index-node (vec children)
-                             (vec op-buf)
-                             cfg))
-          'hitchhiker.tree.messaging.InsertOp
-          msg/map->InsertOp
-          'hitchhiker.tree.messaging.DeleteOp
-          msg/map->DeleteOp
-          'hitchhiker.tree.Config
-          tree/map->Config
+  (let [tree-datanode 'hitchhiker.tree.DataNode
```

Comment:
Why do you bind these names in the let before?

Reply:
This was from the namespace splitting and used for more convenience with the reader conditionals. I did them there for some reuse. But now that you mention it I think it is better having the full context locally.